neon_utils/skills/neon_skill.py

```diff
@@ -90,6 +99,9 @@ def __init__(self, name=None, bus=None, use_settings=True):
         self.neon_core = True
         self.actions_to_confirm = dict()
 
+        self.skill_mode = self.user_config.content.get('util_params').get('mode') or DEFAULT_SPEED_MODE
```

Comment:
`skill_mode` is a little generic.. maybe `speed_mode` if this describes a mode or `intent_timeout` if this is a duration?

Reply:
Sure, that makes sense